phpBB/phpbb/attachment/upload.php

```diff
@@ -326,10 +327,36 @@ protected function check_attach_quota()
 	/**
 	 * Check if there is enough free space available on disk
 	 *
-	 * @return bool True if disk space is available, false if not
+	 * @return bool True if disk space is available or can't get it, false if not
```

Comment:
What do you mean with "can't get it"?

Reply:
If the available space method is not available.
For example AWS has no limit, so the method won't be available and will always allow to upload files.

Reply:
In other words, if you can't get the free space, the file will be uploaded

Reply:
Here is the same https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/phpbb/attachment/upload.php#L299
if disk_free_space fails, the file will be uploaded

Reply:
Maybe then change it to say something like:
`True if disk space is available or not limited, false if not`